signer/tests/integration/rbf.rs

```diff
@@ -323,7 +328,7 @@ pub fn transaction_with_rbf(
             utx.requests
                 .iter()
                 .filter_map(RequestRef::as_withdrawal)
-                .zip(utx.tx.output.iter().skip(1))
+                .zip(utx.tx.output.iter().skip(2))
```

Comment:
I assume this skip 2 is to skip the magic bytes? Might be worth adding a comment to that effect.

Reply:
Oh, right yeah. The outputs always follow a particular order: the first two outputs are the signers' UTXO and the `OP_RETURN` outputs, and then follows outputs servicing withdrawal requests. This test code only cares about the withdrawal request outputs, I'll make sure to add a comment here to that effect.